Form/Type/ModelAutocompleteType.php

```diff
@@ -92,6 +92,10 @@ public function buildView(FormView $view, FormInterface $form, array $options)
         $view->vars['template'] = $options['template'];
 
         $view->vars['context'] = $options['context'];
+
+        // add button
+        $view->vars['btn_add'] = $options['btn_add'];
+        $view->vars['btn_catalogue'] = $options['btn_catalogue'];
```

Comment:
For bonus points refactor this method to avoid all this repetition.

Reply:
Don't really unserstand.
I made a copy/paste of the ModelType class for this part

Reply:
If you want you could have 

```php
foreach ([
    // other keys, see above
    'btn_add',
    'btn_catalogue',
] as $passthroughOption) {
    $view->vars[$passthroughOption] = $options[$passthroughOption];
}
```

Reply:
Don't know if this is more readable @greg0ire

Reply:
Can you tell if all variables names are exactly like all option names with the old version? No. You have to carefully check all of them to be sure. Also, it leaves more room for copy/paste/adapt mistakes.

Reply:
As I said, it's as a bonus, if you want to. Can be in a separate PR, in a separate commit, or not at all, you decide.

Reply:
Done